handbook/product/README.md

```diff
@@ -393,7 +393,11 @@ Directly Responsible Individuals (DRI) engage in the ritual(s) below at the freq
 | Bug review (CX)                               | Weekly (Tuesday)         | The product manager meets with the engineering manager and a product quality engineer to ensure synchronization on bug prioritization and actions preceding CX estimation sessions.                                                                                                         | Mo Zhu            |
 | Churned bug review                            | Sprintly (Wednesday)     | The Head of Product meets with the group product managers to examine churned bugs and categorize them as either continue, needs prioritization, or de-prioritize.                                                                                                                                     | Mo Zhu            |
 | Churned bug clean-up | Sprintly | Following the bug de-prioritization meeting, Product Operations completes the churned bug clean-up, ensuring all necessary follow-up tasks are actioned to classify bugs as continue, needs prioritization, or de-prioritized. This may include relocating bug tickets, adjusting labels, communicating with stakeholders, writing documentation, and closing issues. | Isabell Reedy |
+Bug round-up | Sprintly (Tuesday) | After each sprint, Product Operations will compile a list of churned bugs, which will detail the issue number, bug specifics, bug age, and an area for product managers to suggest subsequent actions.| Isabell Reedy |
+Quality check | Daily | Every day, Product Operations will review the "settled" column on the drafting board to ensure all product action items are complete. | Isabell Reedy |
 
+### Product calendar 
```

Comment:
This is neat, but design reviews like all scrum rituals need to stay on their original calendars.  Feature fest needs to stay on the office hours calendar, etc.

It's fine to have this calendar exist and use the hack of referencing its special email address as an attendee, but we cannot move events to different calendars.  The events on the scrum calendars and office hours calendars are intended to stay there, so they are visible to all fleeties from their first few days at the company.


For example, the #g-cx calendar is the primary calendar for all contributors in that group.  Folks in that group see that as their first team, not their department.

For example, the office hours calendar is a way for all fleeties to join and submit and advocate for feature requests.  It is designed explicitly not to be limited to the product team, and to be discoverable on the office hours calendar.  The office hours calendar owns it and is its primary calendar.

<img width="301" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/6a29790b-a907-4e8d-af6d-93fc66bd7aa5">

Reply:
@Sampfluger88 thanks for using your super permissions to help me get these last couple changed back to the CX, MDM and Office Hours calendars. I ran into an obstacle trying to change the hosts.